client/blocks/image-editor/test/image-editor-canvas.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+class ImageEditorCropMock extends Component {
+	constructor( props ) {
+		super( props );
+	}
+	render() {
+		return <dfn />;
+	}
+}
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
+
+	let ImageEditorCanvas,
+		isImageEditorImageLoaded,
+		wrapper;
+
+	useSandbox( ( sandbox ) => {
+		isImageEditorImageLoaded = sandbox.stub().returns( true );
+	} );
+
+	useMockery( mockery => {
+		mockery.registerMock( 'state/ui/editor/image-editor/selectors', {
+			isImageEditorImageLoaded
+		} );
```

Comment:
Ah, no. Good pick up. I wanted to be able to stub the return value for some esoteric test which I abandoned and after which I failed to clean up. 👍